assets/src/domain/eventEditor/ui/datetimes/datesList/editable/EditableName.tsx

```diff
@@ -24,16 +24,13 @@ const EditableName: React.FC<EditableNameProps> = ({ className, entity: datetime
 	);
 
 	const dateName = datetime.name ? datetime.name : __('Edit title...');
+	const editableProps = {
+		className,
+		onSubmit,
+		defaultValue: dateName,
+	};
 
-	return view === 'table' ? (
-		<InlineEditText className={className} ellipsis={false} onChange={onChangeName}>
-			{dateName}
-		</InlineEditText>
-	) : (
-		<InlineEditHeading level={3} className={className} onChange={onChangeName}>
-			{dateName}
-		</InlineEditHeading>
-	);
+	return view === 'table' ? <InlineEditText {...editableProps} /> : <InlineEditHeading {...editableProps} />;
```

Comment:
ya maybe this can be something like:

```suggestion
	return view === 'table' ? <InlineEditText {...editableProps} /> : <h3><InlineEditText {...editableProps} /></h3>;
```

Reply:
I've used `as` prop, this being more concise way.
`as` prop being part of `Box`, as per docs:
```
Editable component composes the Box component so can pass all Box props.
```

Reply:
https://github.com/eventespresso/event-espresso-core/pull/2695/commits/7406b28b1f4de7767856e97379061a65d2113838

Reply:
https://github.com/eventespresso/event-espresso-core/pull/2695/commits/003e4cb6f62224af01e1b9bf6c0f124ef5e34012